Sources/SwiftSyntaxBuilder/gyb_generated/DeclBuildables.swift

```diff
@@ -0,0 +1,1082 @@
+      rightSquare: rightSquare
+    )
+  }
+}
+
+extension ClosureExpr {
+  public init(
+    leftBrace: TokenSyntax = Tokens.`leftBrace`,
+    signature: ClosureSignature? = nil,
+    @CodeBlockItemListBuilder statementsBuilder: () -> CodeBlockItemList = { .empty },
+    rightBrace: TokenSyntax = Tokens.`rightBrace`
+  ) {
+    self.init(
+      leftBrace: leftBrace,
+      signature: signature,
+      statements: statementsBuilder(),
+      rightBrace: rightBrace
+    )
+  }
+}
+
+extension FunctionCallExpr {
+  public init(
+    calledExpression: ExprBuildable,
+    leftParen: TokenSyntax? = Tokens.`leftParen`,
+    @TupleExprElementListBuilder argumentListBuilder: () -> TupleExprElementList = { .empty },
+    rightParen: TokenSyntax? = Tokens.`rightParen`,
+    trailingClosure: ClosureExpr? = nil,
+    @MultipleTrailingClosureElementListBuilder additionalTrailingClosuresBuilder: () -> MultipleTrailingClosureElementList = { .empty }
+  ) {
+    self.init(
+      calledExpression: calledExpression,
+      leftParen: leftParen,
+      argumentList: argumentListBuilder(),
+      rightParen: rightParen,
+      trailingClosure: trailingClosure,
+      additionalTrailingClosures: additionalTrailingClosuresBuilder()
+    )
+  }
+}
+
+extension SubscriptExpr {
+  public init(
+    calledExpression: ExprBuildable,
+    leftBracket: TokenSyntax = Tokens.`leftSquareBracket`,
+    @TupleExprElementListBuilder argumentListBuilder: () -> TupleExprElementList = { .empty },
+    rightBracket: TokenSyntax = Tokens.`rightSquareBracket`,
+    trailingClosure: ClosureExpr? = nil,
+    @MultipleTrailingClosureElementListBuilder additionalTrailingClosuresBuilder: () -> MultipleTrailingClosureElementList = { .empty }
+  ) {
+    self.init(
+      calledExpression: calledExpression,
+      leftBracket: leftBracket,
+      argumentList: argumentListBuilder(),
+      rightBracket: rightBracket,
+      trailingClosure: trailingClosure,
+      additionalTrailingClosures: additionalTrailingClosuresBuilder()
+    )
+  }
+}
+
+extension ExpressionSegment {
+  public init(
+    backslash: TokenSyntax = Tokens.`backslash`,
+    delimiter: String?,
+    leftParen: TokenSyntax = Tokens.`leftParen`,
+    @TupleExprElementListBuilder expressionsBuilder: () -> TupleExprElementList = { .empty },
+    rightParen: TokenSyntax = Tokens.`stringInterpolationAnchor`
+  ) {
+    self.init(
+      backslash: backslash,
+      delimiter: delimiter == nil ? nil : Tokens.rawStringDelimiter(delimiter!),
+      leftParen: leftParen,
+      expressions: expressionsBuilder(),
+      rightParen: rightParen
+    )
+  }
+}
+
+extension StringLiteralExpr {
+  public init(
+    openDelimiter: String?,
+    openQuote: TokenSyntax,
+    @StringLiteralSegmentsBuilder segmentsBuilder: () -> StringLiteralSegments = { .empty },
+    closeQuote: TokenSyntax,
+    closeDelimiter: String?
+  ) {
+    self.init(
+      openDelimiter: openDelimiter == nil ? nil : Tokens.rawStringDelimiter(openDelimiter!),
```

Comment:
I have moved default value down here? 
What do you think?

Reply:
Sorry, only saw these comments now. I was looking through your changed files and didn’t open the generated file. Sorry for that.

I don’t quite understand what you’re trying to say but the code looks good to me.

Reply:
It's okay! 

Not sure what I have thought. Forgot it. Sorry 😆
Will close